connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/RestServerTest.java

```diff
@@ -180,19 +176,26 @@ public void testOptionsDoesNotIncludeWadlOutput() {
         server = new RestServer(workerConfig);
         server.start(new HerderProvider(herder), herder.plugins());
 
-        Response response = request("/connectors")
-            .accept(MediaType.WILDCARD)
-            .options();
-        Assert.assertEquals(MediaType.TEXT_PLAIN_TYPE, response.getMediaType());
+        HttpRequest request = new HttpOptions("/connectors");
```

Comment:
Why not make this `HttpOptions request` to avoid the casting later in the method?

Reply:
Good catch @rhauch , updated!